tests/rip7560/test_send_failed.py

```diff
@@ -275,3 +278,42 @@ def test_insufficient_pre_transaction_gas(tx_7560):
         "insufficient ValidationGasLimit(30000) to cover PreTransactionGasCost(31000)",
         -32000,
     )
+
+
+# pylint: disable=duplicate-code
+def test_overflow_block_gas_limit(w3: Web3, tx_7560: TransactionRIP7560):
```

Comment:
this is only partial check.
need to test that entire block can overflow the block gas limit
that is, after including AATXs, legacy TXs should be limited to the excess gas
(e.g: add N legacy TXs to the pull, and make sure that only smaller "n" are included, depending on gas left

Reply:
Well, I know from the implementation that the block gas pool is a globally shared object so I don't expect any issues there, so this test would not be very different from this one. I suggest splitting these two checks into separate issues as there are diminishing returns in testing more elaborate combinations.